lib/streamlit/elements/map.py

```diff
@@ -469,10 +467,7 @@ def marshall(
     pydeck_json: str,
     use_container_width: bool,
 ) -> None:
-    json_bytes = pydeck_json.encode("utf-8")
-    id = hashlib.md5(json_bytes, **HASHLIB_KWARGS).hexdigest()
-
     pydeck_proto.json = pydeck_json
     pydeck_proto.use_container_width = use_container_width
 
-    pydeck_proto.id = id
+    pydeck_proto.id = ""
```

Comment:
nit: I think `""` is the default value of the proto string field. So setting it to `""` might be redundant, although it is also explicit so we might want to keep it. But just wanted to point it out (in case we want to add a comment or so)

Reply:
Same as above, this will soon be a legit value, so will just leave for the moment!